### PR TITLE
Fixed bugs on Win.

### PR DESCRIPTION
**Pull request by Shawn-Guo-CN**

### CODE REVIEW

File: spikely/pipeman/pipeman.py [hunk 1]
```diff
@@ -50,7 +52,11 @@ def append(self, text):
         self.output.append(text)
 
     def stdout_ready(self):
-        text = bytearray(self.process.readAllStandardOutput()).decode()
+        if sys.platform.startswith('win'):
+            text = bytearray(self.process.readAllStandardOutput()).decode(os_encoding)
+        else:
+            text = bytearray(self.process.readAllStandardOutput()).decode()
+
```
Comment (rogerhurwitz) [inline spikely/pipeman/pipeman.py]: Does it make sense to use an os specific os_encoding parameter for both Linux and Windows?  decode() without a parameter defaults to utf-8 I believe.  Maybe we should do the same thing you are doing for Windows on Linux as well, and remove the if/else logic.

Comment (Shawn-Guo-CN) [reply spikely/pipeman/pipeman.py]: Yes, it makes sense. Actually, that's what I did in the first fix-up. But, as I haven't tested it on Linux, I left this part to a safe zone.

IMO, all if/else could be deleted if the lines for Win could run on Linux.

File: spikely/pipeman/pipeman.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import sys
+from ctypes import cdll
+os_encoding = 'cp' + str(cdll.kernel32.GetACP())
```
Comment (rogerhurwitz) [inline spikely/pipeman/pipeman.py]: Will this work on a Linux system?

Comment (Shawn-Guo-CN) [reply spikely/pipeman/pipeman.py]: My bad, it won't. I just added another if/else.

**Comment by colehurwitz** [comment]
It works on my linux machine.

**Review by rogerhurwitz — APPROVED** [review]
Excellent - much cleaner.  Nice work!

**Pull request merged**